src/main/util.ts

```diff
@@ -37,5 +39,23 @@ export const mkdir = (dirPath: string) => {
   }
 };
 
+export const handleOSQuery: () => OperatingSystems | null = () => {
+  const isDarwin = os.platform() === 'darwin';
+  const isWindows = os.platform() === 'win32';
+  const isLinux = os.platform() === 'linux';
+
+  if (isDarwin) {
```

Comment:
Nitpick - could just map OS identifier strings to enum values and return that, with null as a default fallback

Reply:
Mapped enum valies with the corresponding OS identifiers here: https://github.com/patrickbrett/mlvet/pull/38/commits/d0e83b46c50d1224dbed8e815b067fd19d3ae308